### PR TITLE
Added info on readme

### PR DESCRIPTION
**Pull request by Mte90**
Based on: 

* https://www.reddit.com/r/discordapp/comments/89n8lf/does_anyone_know_how_to_use_ratismals_uno_bot/
* https://github.com/Ratismal/UNO/issues/11

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -94,6 +115,55 @@ You may selfhost (AKA run your own instance of) this bot under the following cir
...
+    "secret": "client secret",
+    "authSecret": "someamazingsecret"
+  },
+  "shard": 0,
+  "prefix": "uno",
+  "shards": {
+    "max": 1,
+    "perCluster": 1
+  }
+}
```
Comment (Ratismal) [inline readme.md:136]: Please move this to a `config.example.json` file in the root directory

File: readme.md [hunk 2]
```diff
@@ -94,6 +115,55 @@ You may selfhost (AKA run your own instance of) this bot under the following cir
...
+  },
+  "development": {
+    "username": null,
+    "password": null,
+    "database": "sqlite::memory:",
+    "host": null,
+    "dialect": "sqlite",
+    "storage": "database.sqlite"
+  }
+}
```
Comment (Ratismal) [inline readme.md:159]: Please move this to a `config/config.example.json` file

File: readme.md [hunk 3]
```diff
@@ -94,6 +115,55 @@ You may selfhost (AKA run your own instance of) this bot under the following cir
...
+```
+
+### Dependencies
+
+After running `npm install` it is required to migrate the new database (after put the configuration files) and run `npx sequelize-cli db:migrate`.
```
Comment (Ratismal) [inline readme.md:164]: I'd recommend using the npm script instead, so you don't have to deal with npx.
```suggestion
After running `npm install` it is required to migrate the new database (after put the configuration files) and run `npm run sequelize db:migrate`.
```

File: readme.md [hunk 4]
```diff
@@ -94,6 +115,55 @@ You may selfhost (AKA run your own instance of) this bot under the following cir
...
+    "dialect": "sqlite",
+    "storage": "database.sqlite"
+  }
+}
+```
+
+### Dependencies
+
+After running `npm install` it is required to migrate the new database (after put the configuration files) and run `npx sequelize-cli db:migrate`.  
+Now you can run the bot with `nodejs index.js`.
```
Comment (Ratismal) [inline readme.md:165]: Please use `node` instead
```suggestion
Now, you can run the bot with `node index.js`.
```

File: readme.md (hunk 3 above)
Comment (Ratismal) [reply readme.md:164]: Additionally, we could make a new run script in package.json to run the migration command directly.

**Review by Ratismal — CHANGES_REQUESTED** [review]
Hey, thanks for making this PR - and sorry it's taken so long for me to get back on this. Just a few request changes before I merge this in.

**Comment by Mte90** [comment]
Updated :-)

**Pull request merged**